jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -24,6 +26,13 @@ def __init__(self, server_url, token, namespace=None, verify_ssl=True, gpu_mode=
     self.api_client = None
     self.namespace = namespace #TODO why do I need to pass namespace?
     self.gpu_mode = gpu_mode
+    self.oapi_client = None
+
+    configuration = kubernetes.client.Configuration()
```

Comment:
Please move this block under the condition on line 38 - after the `self.api_client = ...` line